include/beast/websocket/stream.hpp

```diff
@@ -896,6 +903,84 @@ class stream : public detail::stream_base
     #endif
     async_close(close_reason const& cr, CloseHandler&& handler);
 
+    /** Send a WebSocket ping frame.
+
+        This function is used to sycnhronously send a ping frame on
```

Comment:
A couple spelling errors in this file (sycnhronously here). Cherry pick this to fix what I found: https://github.com/seelabs/Beast/commit/db564b94bc06410548a833c3f16a0c9dfe5e712d